Delver/Assets/Source/Gameplay/Abilities/Ability_SpecialAttack.cs

```diff
@@ -112,7 +112,7 @@ protected void SpawnColliderLayer(int layer)
                 addedTime = Time.time
             };
 
-            newEntry.spawnedCollider.InitFromAttack(abilityUser, OnColliderEnter, OnColliderExit, ValidateHit);
+            newEntry.spawnedCollider.InitFromAttack(abilityUser, OnColliderEnter, OnColliderExit, ValidateHit, this.activatedDirection, this.colliderSpeed);
```

Comment:
Personal style preference, "this" reference is not needed. Its obvious enough that your in the object and the "this" is implicit